src/components/modal/OrderModal.js

```diff
@@ -9,8 +9,12 @@ export default function OrderModal({ onTrade, onClose, isTrade, children }) {
       <Dimmed onClick={onClose} />
       <CheckModalWrapper>
         <Div>{children}</Div>
-        <Button onClick={isTrade ? onTrade : onClose}>확인</Button>
-        <Button onClick={onClose}>취소</Button>
+        <Button data-testid="confirm-btn" onClick={isTrade ? onTrade : onClose}>
```

Comment:
```suggestion
        <Button data-testid="confirm-button" onClick={isTrade ? onTrade : onClose}>
```
지금 작성되어 있는 코드로는 button을 줄이지 않고 작성한 코드가 더 많은 거 같아요 ! 일관성 있게 button으로 작성해주시면 더 좋을 거 같습니다 !

Reply:
넵 수정했습니다